ddtrace/sampler.py

```diff
@@ -199,6 +200,34 @@ def __init__(
             )
             self.default_sampler = SamplingRule(sample_rate=default_sample_rate)
 
+    def parse_from_env_variable(self):
```

Comment:
```suggestion
    def _parse_from_env_variable(self):
```

we want to make this helper private.

Reply:
we might also want to have "rules" in the method name, like `_parse_rules_from_env_variable()`, wdyt?

Reply:
👍